apps/client/src/index.tsx

```diff
@@ -1,9 +1,19 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import { RouterProvider } from 'react-router-dom';
+import { applyMode, Mode } from '@cloudscape-design/global-styles';
+import { Amplify } from 'aws-amplify';
+
 import '@cloudscape-design/global-styles/index.css';
+import { router } from './router';
+
+const awsResources = (global as any).awsResources;
+
+Amplify.configure({
+  ...awsResources,
+});
 
-import { router } from './router/router';
+applyMode(Mode.Dark);
```

Comment:
will switching this to Mode.Light just "work" or will we need to do styling tweaks?

Reply:
It will just work. :)